packages/eslint-config-universe/package.json

```diff
@@ -56,8 +57,8 @@
   },
   "devDependencies": {
     "@expo/spawn-async": "^1.5.0",
-    "eslint": "^6.6.0",
-    "jest": "^24.9.0",
+    "eslint": "^6.8.0",
```

Comment:
I couldn't comment on L55 above because of https://github.com/isaacs/github/issues/284, but does the eslint peer dependency version on L55 still hold given this change?

Reply:
Yes, it does. I just updated this version so that internally we get the latest version of the linter, but `prefer-const` is not new (is really old actually).